modules/installation-and-upgrade/pages/container-deployment/prepare-micro-host.adoc

```diff
@@ -0,0 +1,114 @@
+. Click [systemitem]``Next`` to continue.
+. Select the registration method.
+  For this example, we will register the server with {scclongform}.
+
+
+[NOTE]
+====
+The {productname} {productnumber} containers are installed as extensions.
+Depending on the specific extension needed from the list below, additional {scclongform} registration codes will be required for each.
+
+* {productname} {productnumber} Server
+* {productname} {productnumber} Proxy
+* {productname} {productnumber} Retail Branch Server
+====
+
+
+[IMPORTANT]
+====
+The {sl-micro} {microversion} entitlement is included within the {productname} entitlement, so it does not require a separate registration code.
+====
+
+. Enter your {scclongform} email address.
+. Enter your registration code for {sl-micro} {microversion}.
+. Click [systemitem]``Next`` to continue.
+. To install a proxy, select the {productname} {productnumber} Proxy extension; to install a server, select the {productname} {productnumber} Server extension ``Checkbox``.
+. Click [systemitem]``Next`` to continue.
+. Enter your {productname} {productnumber} extension registration code.
+. Click btn:[Next] to continue.
+. On the [systemitem]``NTP Configuration`` page click btn:[Next].
+. On the [systemitem]``Authentication for the System`` page enter a password for the root user.
+  Click btn:[Next].
+. On the [systemitem]``Installation Settings`` page click btn:[Install].
+
+This concludes installation of {sl-micro} {microversion} and {productname} {productnumber} as an extension.
+
+
+
+.Procedure: Registration from the Command Line (Optional)
+
+If you added {productname} {productnumber} as an extension during {sl-micro} {microversion} installation then you can skip this step. However, optionally you may skip registration during {sl-micro} {microversion} installation by selecting the btn:[Skip Registration] button.
+This section provides steps on registering your products after {sl-micro} {microversion} installation.
+
+[NOTE]
+====
+The following steps register a {productname} {productnumber} extension with the {x86_64} architecture and thus require a registration code for the {x86_64} architecture.
+To register {arm} or {s390x} architectures use the correct registration code.
+====
+
+. List available extensions with the following command:
+
++
+
+----
+transactional-update --quiet register --list-extensions
+----
+
+. From the list of available extensions, select the one you wish to install
+
+.. If installing the Server, use your {productname} Server Extension {productnumber} x86_64 registration code with following command:
+
++
+
+----
+transactional-update register -p SUSE-Manager-Server/5.1/x86_64 -r <reg_code>
+----
+.. If installing the Proxy, use your {productname} Proxy Extension {productnumber} x86_64 registration code with following command:
+
++
+
+----
+transactional-update register -p SUSE-Manager-Proxy/5.1/x86_64 -r <reg_code>
+----
```

Comment:
and transactional-update register -p Multi-Linux-Manager-Proxy/5.1/x86_64 -r <reg_code>